src/cr/cube/crunch_cube.py

```diff
@@ -1059,6 +1059,33 @@ def percentages(self, axis=None):
         '''
         return self.proportions(axis) * 100
 
+    def population_counts(self, population_size):
+        '''Get the population counts for crunch cube values relative to the total population size estimate.
+
+        This function calculates the population counts for crunch cube values. The
+        population counts are based on the values of the 'proportions'.
+
+        Args
+            population_size (int): Estimated total population size
+
+        Returns
+            (nparray): Calculated array of crunch cube population counts.
+
+        Example:
+            >>> cube = CrunchCube(fixt_cat_x_cat)
+            np.array([
```

Comment:
@kwypchlo It's not clear what this output is the result of? I'm guessing it's the `cube.as_array()`. We can either add that line, or make a `__repr__` method work properly :) You decide :)

Reply:
I would test `cube.as_array()` explicitly